src/ExpressionEvaluator/CSharp/Test/ExpressionCompiler/DeclarationTests.cs

```diff
@@ -504,7 +505,7 @@ static void M<T>(object x)
                     out error,
                     testData);
 
-                Assert.Equal(testData.Methods.Count, 1);
+                Assert.Equal(testData.Methods.Where(m => !m.Key.IsImplicitlyDeclared).Count(), 1);
```

Comment:
>testData.Methods.Where(m => !m.Key.IsImplicitlyDeclared) [](start = 29, length = 56)

Perhaps add a `GetExplicitlyDeclaredMethods()` extension method to `CompilationTestDataExtensions`.

Reply:
I'm tempted to leave this one as-is.

---
In reply to: [214453975](https://github.com/dotnet/roslyn/pull/29180#discussion_r214453975) [](ancestors = 214453975)